services/Skyra.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace Skyra.Tests
+{
+	public class Tests
+	{
+		[SetUp]
+		public void Setup()
+		{
+		}
+
+		[Test]
+		public void Test1()
```

Comment:
we'll eventually write actual tests (@Favna can help me scream at kyra to add tests as we go)